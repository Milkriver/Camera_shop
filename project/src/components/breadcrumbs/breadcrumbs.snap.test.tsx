import {render} from '@testing-library/react';
import { createMemoryHistory } from 'history';
import HistoryRouter from '../history-router/history-router';
import Breadcrumbs from './breadcrumbs';

const history = createMemoryHistory();
const mockName = '';


describe('Component: Breadcrumbs', () => {
  it('should render correctly', () => {
    const {container} = render(
      <HistoryRouter history={history}>
        <Breadcrumbs name = {mockName} />
      </HistoryRouter>
    );

    expect(container).toMatchSnapshot();
  });
});
