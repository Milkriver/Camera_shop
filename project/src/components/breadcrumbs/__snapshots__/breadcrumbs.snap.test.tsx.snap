// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Breadcrumbs should render correctly 1`] = `
<div>
  <div
    class="breadcrumbs"
  >
    <div
      class="container"
    >
      <ul
        class="breadcrumbs__list"
      >
        <li
          class="breadcrumbs__item"
        >
          <a
            class="breadcrumbs__link"
            href="/"
          >
            Главная
            <svg
              aria-hidden="true"
              height="8"
              width="5"
            >
              <use
                xlink:href="#icon-arrow-mini"
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
